docs/dx_tables.md

```diff
@@ -3,14 +3,660 @@
+described as a sequence of one or more Table B mnemonics which
+correspond to basic data values (e.g. pressure, temperature, humidity,
+etc.).
+
+<br>
+
+The entire sequence of data values that constitute a particular type
+of data subset is fully and unambiguously defined.
+
+<br>
+
+Mnemonics never themselves appear within actual BUFR messages.  Their
+only purpose is to make it easier for users to interact with the
+software by providing descriptive names to represent individual data
+values. They are more intuitive than FXY numbers (described below),
+which are the prescribed method within actual BUFR messages.
+
+## DX BUFR Tables File
+
+A DX BUFR tables file consists of three distinct sections.  Each
+section contains one or more lines of 80 characters in length, and
+where a "*" as the first character of a line indicates that that
+entire line is a comment.
+
+1. In the [first section](#section1), all Table A, B and D mnemonics
+that are to be used within the file are initially declared, assigned a
+unique FXY number, and given a short, free-form text description.
+
+2. In the [second section](#section2), all previously-declared Table A
+and Table D mnemonics are actually defined as a sequence of one or
+more Table B (or other Table D) mnemonics.
+
+3. In the [third section](#section3), all previously-declared Table B
+mnemonics are defined in terms of their scale factor, reference value,
+bit width, and units.
+
+<br>
+
+<div id="section1">
+### Section 1
+
+The first section of a BUFR tables file is where all Table A, B and D
+mnemonics are initially declared, assigned a unique FXY number, and
+given a short free-form text description. Mnemonics may contain any
+combination of uppercase letters and numbers (or, in certain special
+cases, a "." character), up to a maximum of 8 characters in length. A
+mnemonic may be declared only once, and each one must correspond to a
+unique FXY number, which itself consists of 6 characters, and where
+the first character (i.e. the "F" component) is an "A" if the mnemonic
+is being declared as a Table A mnemonic, "3" if the mnemonic is being
+declared as a Table D mnemonic, and "0" if the mnemonic is being
+declared as a Table B mnemonic. Otherwise, the remainder of the FXY
+number must be all digits, with the next 2 characters (i.e. the "X"
+component) as a number between 00 and 63, and the final 3 characters
+(i.e. the "Y" component) as a number between 001 and 255. Readers who
+are more familiar with BUFR will immediately recognize these F, X, and
+Y values as those that are defined within the [official documentation
+of the BUFR code form](@ref manual).
+
+<br>
+
+By international convention, a mnemonic should not be given an X value
+between 00 and 47 along with a Y value between 001 and 191 unless that
+mnemonic, when subsequently defined, corresponds exactly to the BUFR
+descriptor having that same FXY number within the [official WMO master
+BUFR tables](@ref wmomstab).
+
+<br>
```

Comment:
FYI, Doxygen has trouble when you split a hyperlink across multiple lines in a markdown file.  It's best to have the entire `[]()` nomenclature on one line.

Reply:
But since you've said you're stepping back now, I can go ahead and push up this minor tweak myself and then approve and merge this PR.